### PR TITLE
Attempting to add recipe for bioconductor-copywriter

### PR DESCRIPTION
**Pull request by holtgrewe**
Includes dependencies:

- bioconductor-chipseq
- bioconductor-copyhelper

* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by holtgrewe** [comment]
@bioconda/core Hm, this failure looks weird to me. Could anyone have a look?

**Comment by johanneskoester** [comment]
Weird, I don't see the actual error causing [this](https://travis-ci.org/bioconda/bioconda-recipes/jobs/209300677#L744). @bioconda/r did anybody see this before?

**Comment by bgruening** [comment]
The error is here:
```
R: No such file or directory
```

https://travis-ci.org/bioconda/bioconda-recipes/jobs/209300677#L603

**Comment by johanneskoester** [comment]
Weird... Does it work locally?

**Comment by holtgrewe** [comment]
@johanneskoester Locally, the following happens:

```
$ conda build recipes/bioconductor-copyhelper
# works
$ conda build recipes/bioconductor-chipseq
BUILD START: bioconductor-chipseq-1.24.0-r3.3.2_0
updating index in: /bioconda/2017-03/miniconda3/conda-bld/linux-64
updating index in: /bioconda/2017-03/miniconda3/conda-bld/noarch
Can't build /bioconda/2017-03/dev/bioconda-recipes/recipes/bioconductor-chipseq due to unsatisfiable dependencies:
The following specifications were found to be in conflict:
  - bioconductor-shortread -> bioconductor-biobase -> bioconductor-biocgenerics >=0.3.2 -> r 3.2.2* -> r-base 3.2.2
  - bioconductor-shortread -> bioconductor-biobase -> bioconductor-biocgenerics >=0.3.2 -> r 3.2.2* -> r-recommended 3.2.2
  - r 3.3.2*
Use "conda info <package>" to see the dependencies for each package.
$ conda build recipes/bioconductor-copywriter
BUILD START: bioconductor-copywriter-2.6.0-r3.3.2_0
updating index in: /bioconda/2017-03/miniconda3/conda-bld/linux-64
updating index in: /bioconda/2017-03/miniconda3/conda-bld/noarch
Missing dependency bioconductor-chipseq, but found recipe directory, so building bioconductor-chipseq first
BUILD START: bioconductor-chipseq-1.24.0-r3.3.2_0
updating index in: /bioconda/2017-03/miniconda3/conda-bld/linux-64
updating index in: /bioconda/2017-03/miniconda3/conda-bld/noarch
Can't build /bioconda/2017-03/dev/bioconda-recipes/recipes/bioconductor-chipseq due to unsatisfiable dependencies:
The following specifications were found to be in conflict:
  - bioconductor-shortread -> bioconductor-biobase -> bioconductor-biocgenerics >=0.3.2 -> r 3.3.1* -> r-recommended 3.3.1
  - r 3.3.2*
Use "conda info <package>" to see the dependencies for each package.

```

**Comment by bgruening** [comment]
@holtgrewe you need to export CONDA_R: https://github.com/bioconda/bioconda-recipes/blob/master/scripts/env_matrix.yml#L6
Or simply pin against r 3.3.1 to avoid this error.

**Comment by holtgrewe** [comment]
OK, with `CONDA_R`, `conda build recipes/bioconductor-chipseq` now works. For `conda build recipes/bioconductor-copywriter`, I now get:

```
ERROR conda.core.link:_execute_actions(335): An error occurred while installing package 'bioconda::bioconductor-dnacopy-1.46.0-r3.3.1_0'.
PaddingError: Placeholder of length '80' too short in package /bioconda/2017-03/miniconda3/conda-bld/bioconductor-copywriter_1489392363941/_b_env/lib/R/library/DNAcopy/libs/DNAcopy.so.
The package must be rebuilt with conda-build > 2.0.

Attempting to roll back.

Can't build /bioconda/2017-03/dev/bioconda-recipes/recipes/bioconductor-copywriter due to environment creation error:
Placeholder of length '80' too short in package /bioconda/2017-03/miniconda3/conda-bld/bioconductor-copywriter_1489392363941/_b_env/lib/R/library/DNAcopy/libs/DNAcopy.so.
The package must be rebuilt with conda-build > 2.0.
```

**Comment by bgruening** [comment]
This can be solved by bumping the build number from bioconductor-dnacopy. You can do this in the same PR.

BG: This is due to an old build of this package with an old conda version, which had a max placeholder length of 80 chars.

**Comment by holtgrewe** [comment]
@bgruening It works locally but fails in Travis... with the `R: No such file or directory` problem

**Comment by holtgrewe** [comment]
@bgruening any idea?

**Comment by johanneskoester** [comment]
When you build locally, does it work with bioconda-utils and --docker?

**Comment by holtgrewe** [comment]
@johanneskoester Let me try again on Monday at work. It's been more than 3 weeks ;)

**Comment by bow** [comment]
Hi @holtgrewe, sorry to (sort-of) bump in your PR, but I'm curious if you've managed to solve this issue?

The reason I am asking is that I am [encountering the same issue](https://travis-ci.org/bow/bioconda-recipes/jobs/229099775) on a bioconductor package I am trying to add [via another PR](https://github.com/bioconda/bioconda-recipes/pull/4621). My symptoms are similar:

* Local build works fine (tried this with conda-build v2.0.7 and v2.1.10).
* `./simulate-travis.py --docker` fails with the same error as Travis.

I see `CONDA_R` has been exported successfully and I've also tried fiddling with the `build.sh` and `meta.yaml` files, all to no avail.

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.
